cloud: add maintenance notification for monitoring features (#17079)
pingcap/docs: tidb-cloud/notification-2024-04-16-monitoring-features-maintenance.md

---
title: 2024-04-16 TiDB Cloud Monitoring Features Maintenance Notification

> summary: Learn about the details of the TiDB Cloud Monitoring features maintenance on April 16, 2024, such as the maintenance window, reason, and impact.

Reviewer: ```suggestion
summary: Learn about the details of the TiDB Cloud monitoring features maintenance on April 16, 2024, such as the maintenance window, reason, and impact.
```

---

> # [2023-08-31] TiDB Cloud Console Maintenance Notification

Reviewer: ```suggestion
# [2024-04-16] TiDB Cloud Monitoring Features Notification
```

> This notification describes the details that you need to know about the [TiDB Cloud console](https://tidbcloud.com/) maintenance on April 16, 2024.

Reviewer: ```suggestion
This notification describes the details that you need to know TiDB Cloud monitoring features maintenance on April 16, 2024.
```

## Maintenance window

- Start time: 2024-04-16 08:00 (UTC+0)
- End time: 2024-04-16 12:00 (UTC+0)
- Duration: 4 hours

## Impact

> ### Impacted region scope:  

Reviewer: ```suggestion
### Affected regions:
	
	During the maintenance window, the monitoring features in the following regions will be affected:

```

> - Monitoring features for TiDB Dedicated clusters in the following regions：

Reviewer: ```suggestion
- TiDB Dedicated clusters：
```

    - Cloud Provider: AWS, Region: Tokyo (ap-northeast-1)
    - Cloud Provider: AWS, Region: N. Virginia (us-east-1)

> - Monitoring features for Serverless clusters in the following regions：

Reviewer: ```suggestion
- TiDB Serverless clusters：
```

    - Cloud Provider: AWS, Region: Tokyo (ap-northeast-1)
    - Cloud Provider: AWS, Region: N. Virginia (us-east-1)

> ### Affected Monitoring features ：

Reviewer: ```suggestion
### Affected monitoring features
```

> **Note:**
>

> > The maintenance only affects the Monitoring features in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.

Reviewer: ```suggestion
> The maintenance only affects the monitoring features in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
```

> - Metrics page will be temporarily unavailable for several short periods（less than 20 mins）.
> - Slow Query page will be temporarily unavailable for several short periods （less than 5 mins）.
> - Metrics Integration with Prometheus, DataDog and NewRelic may have breakpoints

Reviewer: ```suggestion
- The **Metrics** page will be temporarily unavailable for several short periods (less than 20 mins).
- The **Slow Query** page will be temporarily unavailable for several short periods (less than 5 mins).
- The metrics integration with Prometheus, DataDog, and NewRelic might have breakpoints.
```

---
title: 2024-04-16 TiDB Cloud Monitoring Features Maintenance Notification
summary: Learn about the details of the TiDB Cloud monitoring features maintenance on April 16, 2024, such as the maintenance window, reason, and impact.
---

> # [2024-04-16] TiDB Cloud Monitoring Features Notification

Reviewer 2: ```suggestion
# [2024-04-16] TiDB Cloud Monitoring Features Maintenance Notification
```

> # [2024-04-16] TiDB Cloud Monitoring Features Notification

Reviewer: ```suggestion
# [2024-04-16] TiDB Cloud Monitoring Features Maintenance Notification
```

> This notification describes the details that you need to know TiDB Cloud [monitoring features](/tidb-cloud/monitor-tidb-cluster.md)maintenance on April 16, 2024.

Reviewer: ```suggestion
This notification describes the details that you need to know TiDB Cloud [monitoring features](/tidb-cloud/monitor-tidb-cluster.md) maintenance on April 16, 2024.
```

## Maintenance window

- Start time: 2024-04-16 08:00 (UTC+0)
- End time: 2024-04-16 12:00 (UTC+0)
- Duration: 4 hours

## Impact

> ### Affected regions:

Reviewer: ```suggestion
### Affected regions
```

> 	During the maintenance window, the monitoring features in the following regions will be affected:

Reviewer: ```suggestion
During the maintenance window, the monitoring features in the following regions will be affected:
```

- TiDB Dedicated clusters：
    - Cloud Provider: AWS, Region: Tokyo (ap-northeast-1)
    - Cloud Provider: AWS, Region: N. Virginia (us-east-1)

- TiDB Serverless clusters：
    - Cloud Provider: AWS, Region: Tokyo (ap-northeast-1)
    - Cloud Provider: AWS, Region: N. Virginia (us-east-1)

### Affected monitoring features

> **Note:**
>
> The maintenance only affects the monitoring features in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.

> - The **Metrics** page will be temporarily unavailable for several short periods (less than 20 mins).
> - The **Slow Query** page will be temporarily unavailable for several short periods (less than 5 mins).

Reviewer: ```suggestion
- The **Metrics** page will be temporarily unavailable for several short periods (each less than 20 mins).
- The **Slow Query** page will be temporarily unavailable for several short periods (each less than 5 mins).
```

> - The **Metrics** page will be temporarily unavailable for several short periods (each less than 20 mins).
> - The **Slow Query** page will be temporarily unavailable for several short periods (each less than 5 mins).

Reviewer: ```suggestion
- The **Metrics** page will be temporarily unavailable for several short periods (each less than 20 minutes).
- The **Slow Query** page will be temporarily unavailable for several short periods (each less than 5 minutes).
```
